nbdime/merging/notebooks.py

```diff
@@ -58,6 +58,7 @@
 
 def notebook_merge_strategies(args):
     strategies = Strategies({
+        "/cells/*/id": "remove",
```

Comment:
Seeking maintainers input: what to do here.

Reply:
I think `remove` would be correct action for "step 1", but we would need to verify that it generates a new id before saving.